docs/Q-and-A.md

```diff
@@ -0,0 +1,649 @@
+
+##  ***Q: "Process operations in accordance with the protocol deterministic ruleset." What does this mean?
+Read this [section](https://identity.foundation/sidetree/spec/#late-publishing) that illustrates a DID owner, Alice, creating forks by creating and anchoring operations in the past that she does not expose to the network. Known as _Late Publishing_ of a DID operation.\
+This type of processing guarantees only one `fork` of a DID’s state history can ever be valid.
+
+##  *Q: What would you see as the main drawback of Sidetree?
+There aren't many. Sidetree has Self-Certifying Identifiers, however they are non-transferable. {Right?}
+_(@henkvancann)_
+
+##  *Q: How can it be one solution, fit for more DIDs? 
+DID Methods based on the Sidetree protocol all share the same identifier format. The unique identifier segment of a Sidetree-based DID, known as the DID Suffix, is derived by using the Hashing Process to generate a hash value from the canonicalized Create Operation Suffix Data Object. The DID Suffix is cryptographically bound to the initial PKI state of the DID, which means Sidetree DIDs are also self-certified.
+
+## *Q: Where you would need something quite different than Sidetree to solve a similar problem?
+In a private situation with no self sovereignty. A Certification Authority in combination with a federated setup could do, like it has done the past decades. But in that SSI become Self Service Identity at best, because the Identifiers are owned and controlled by a third party.
+_(@henkvancann)_
+
+## **Q: How does Sidetree scale
+This image speaks a thousand words:\
+<img src="./images/Sidetree-file-structure.png" alt="Sidetree file structure" border="0" width="500">
+Sidetree is a lean and mean structure. The content of Sidetree mainly consists of JSON files of hashes, signatures and other cryptographical proofs and pointers. And it connects to other highly scaleable technologies, like IPFS and blockchains.\
+<img src="./images/sidetree-logo.png" alt="Sidetree logo" border="0" width="100" align="right">
+The importance of this structure for scaleability is rewarded in the Sidetree-logo.
+
+## ***Q: How does Sidetree keep data internally consistent over nodes?
+In brief: By using **delta-based** Conflict-Free Replicated Data Types. _(@henkvancann)_
+
+Despite early availability on the market, `BX` tools did not gain much user appreciation because of semantic issues. A user should clearly understand the behavior of synchronization procedures implemented by a tool. The majority of algebraic BX frameworks are _state-based_. Synchronizing operations take the states of models before and after update as input, and produce new states of models as output. This design assumes that model alignment, i.e., discovering relations (deltas) between models,
```

Comment:
nit: a link to the abbreviation section would be helpful for BX

Reply:
done